functions/googleGroupsManager.test.js

```diff
@@ -0,0 +1,70 @@
+const authenticator = require('./authenticator')
+const googleGroupsManager = require('./googleGroupsManager');
+
+// gMail API constants.
+const {google} = require('googleapis');
+const DOMAIN = 'identity-sre.com';
```

Comment:
Can we make it to be added as a parameter to be appended by the user, so that the user doesn't have to modify the source code if their testing environment is different?

Reply:
Sure, I'll create a config file.

Reply:
config.json <- that's the file with configurations.